src/parser/Parser.spec.ts

```diff
@@ -726,6 +726,21 @@ describe('parser', () => {
             expect(fn.annotations[0].call).to.be.instanceof(CallExpression);
         });
 
+        it('attaches annotations to a class', () => {
+            let { statements, diagnostics } = parse(`
+            @meta1
```

Comment:
Can you indent this stuff properly? Everything but the `end class` needs indented once more